tests/Aggregation/BucketSelectorTest.php

```diff
@@ -46,9 +46,12 @@ public function testToArray(): void
      */
     public function testMaxAggregation(): void
     {
+        $this->_checkVersion('7.2');
```

Comment:
Cannot we do the same you've done later for `DateHistogramTest::testDateHistogramAggregation()`, I mean checking the version of elasticsearch and using `setInterval()` or `setFixedInterval()` depending the version?